src/mds/MDSMap.cc

```diff
@@ -687,6 +678,11 @@ MDSMap::availability_t MDSMap::is_cluster_available() const
     return AVAILABLE;
   } else {
     // Nothing indicating we were stuck, but nobody active (yet)
-    return TRANSIENT_UNAVAILABLE;
+    //return TRANSIENT_UNAVAILABLE;
+
+    // Because we don't have standbys in the MDSMap any more, we can't
+    // reliably indicate transient vs. stuck, so always say stuck so
+    // that the client doesn't block.
+    return STUCK_UNAVAILABLE;
```

Comment:
This impacts... whether ceph-fuse sits around waiting or errors out when it starts up?

How would we address this instead?  Put a standby counter in the MDSMap, or do it external to the mdsmap in ceph-fuse (or any other Client) during the mount process?